assets/component-cart-notification.css

```diff
@@ -58,8 +58,8 @@
 
 .cart-notification__heading .icon-checkmark {
   color: rgb(var(--color-foreground));
-  margin-right: calc(var(--font-heading-scale) * 1rem);
-  width: calc(var(--font-heading-scale) * 1.3rem);
```

Comment:
Curious as to why we are removing this. Shouldnt the heading size affect this?

Reply:
I've added `text-body` class to `cart-notification__heading` element. So the icon rendered as child of `cart-notification__heading` should now scale with the body font size-setting and not the heading font-size setting.
```
<h2 class="cart-notification__heading caption-large text-body">{%- render 'icon-checkmark' -%} {{ 'general.cart.item_added' | t }}</h2>
```